airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -2988,6 +3033,7 @@ def __init__(
         batch: dict | Batch,
         batch_id: str | None = None,
         request_id: str | None = None,
+        num_retries_if_resource_is_not_ready: int | None = None,
```

Comment:
Can we just use  `num_retries_if_resource_is_not_ready: int =0`?

Reply:
I have updated it, thanks!